src/web/mjs/connectors/LianScans.mjs

```diff
@@ -8,13 +8,33 @@ export default class LianScans extends WordPressMangastream {
         super.label = 'LIAN';
         this.tags = [ 'manga', 'indonesian', 'scanlation' ];
         this.url = 'https://www.lianscans.my.id';
-        this.path = '/manga-list/';
+        this.path = '/manga/';
 
-        this.querMangaTitleFromURI = 'div.series-title h2';
-        this.queryMangas = 'div.mangalist-blc ul li a.series';
-        this.queryChapters = 'div.series-chapter ul.series-chapterlist li div.flexch-infoz a';
-        this.queryChaptersTitle = 'span';
-        this.queryChaptersTitleBloat = 'span > span.date';
-        this.queryPages = 'div.reader-area img[src]:not([src=""])';
+        this.queryMangas = 'div.listupd div.bs div.bsx > a';
+        this.queryChapters = 'div.eplister div.chbox div.eph-num a';
+        this.queryPages = 'div.readerarea img[src]:not([src=""])';
+        this.queryChaptersTitle = 'div.eplister div.chbox div.eph-num a span.chapternum';
+        this.querMangaTitleFromURI = 'div.infox h1.entry-title';
     }
-}
\ No newline at end of file
+
+    async _getMangas() {
+        let mangaList = [];
+        for (let page = 1, run = true; run; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangas.length > 0 ? mangaList.push(...mangas) : run = false;
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        const uri = new URL('/manga/?page=' + page, this.url);
+        const request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.listupd div.bs div.bsx > a');
```

Comment:
Why not using this.queryMangas which you defined?

Reply:
there is this to solve. 
        let data = await this.fetchDOM(request, 'div.listupd div.bs div.bsx > a');
could be replaced by  
        let data = await this.fetchDOM(request, this.queryMangas);